### ISSUE TITLE
The link generated to login into template/client not working

### ISSUE THREAD

**Issue by mmbmha** [it-projects-llc/odoo-saas-tools#625]
I created a plan successfully but when I try to generate a link to log in it will give me 404 page.
I already installed the 'Local IP for OAuth requests' module but it is still not working 

generated URL sample"http://t2.odoo.local/saas_client/edit_database?access_token=X4QSBuiMdBOprcsCB9p1815lUSZRMr&state=%7B%22host%22%3A+%22t2.odoo.local%22%2C+%22public_url%22%3A+%22http%3A%2F%2Ft2.odoo.local%2F%22%2C+%22d%22%3A+%22t2%22%2C+%22client_id%22%3A+%22ac553232-6230-11e7-a94d-000c29f572b2%22%7D" 

please help

**Comment by 360org** [comment]
wrong url check your auth provider

**Comment by mmbmha** [comment]
I followed the instructions and used Saas as my provider

**Comment by 360org** [comment]
check again it must be somethings wrong

**Comment by mmbmha** [comment]
<img width="977" alt="screen shot 2017-07-10 at 10 04 43 am" src="https://user-images.githubusercontent.com/18324351/28006287-5670caf0-6557-11e7-9a3b-19f40395ca6f.png">
can you please tell me what is wrong?

**Comment by 360org** [comment]
is that your sytem runing local?

**Comment by 360org** [comment]
have you revert proxy to local yet?

**Comment by mmbmha** [comment]
Yes. when I try to login to the server using Saas it works fine but it does not work when i login to the template nor the client

**Comment by 360org** [comment]
proxy not revert to client yet. try to check your nginx config

**Comment by mmbmha** [comment]
which part exactly? :|

**Comment by njeudy** [comment]
@mmbmha you should have odoo database if you go to http://t2.odoo.local .. think you do not have entry in your proxy to point this (like *.odoo.local or so) 

This entry should point to saas-server ip where you create plan

What is the version of odoo ?
